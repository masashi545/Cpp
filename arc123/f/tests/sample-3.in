2 1 10
5 15
