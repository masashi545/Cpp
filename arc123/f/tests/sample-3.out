8 3 10 7 4 9 5 6 7 8 9
